diff --git a/node_modules/@navikt/navspa/dist/navspa.js b/node_modules/@navikt/navspa/dist/navspa.js
index d977ba8..3f9a605 100644
--- a/node_modules/@navikt/navspa/dist/navspa.js
+++ b/node_modules/@navikt/navspa/dist/navspa.js
@@ -48,8 +48,8 @@ var React = __importStar(require("react"));
 var ReactDOM = __importStar(require("react-dom"));
 var Feilmelding = __importStar(require("./feilmelding"));
 var utils_1 = require("./utils");
-exports.scope = global['NAVSPA'] = global['NAVSPA'] || {}; // tslint:disable-line
-exports.scopeV2 = global['NAVSPA-V2'] = global['NAVSPA-V2'] || {}; // tslint:disable-line
+exports.scope = globalThis['NAVSPA'] = globalThis['NAVSPA'] || {}; // tslint:disable-line
+exports.scopeV2 = globalThis['NAVSPA-V2'] = globalThis['NAVSPA-V2'] || {}; // tslint:disable-line
 exports.exportEvent = 'NAVSPA-eksporter';
 function eksporter(name, component) {
     exports.scope[name] = function (element, props) {
